js/read_exif_workflow.js

```diff
@@ -0,0 +1,59 @@
+import { app } from '../../scripts/app.js'
+import { ExifReader } from './lib/exif-reader.js' // https://github.com/mattiasw/ExifReader v4.26.2
+
+app.registerExtension({
+    name: "ComfyUI-Image-Saver",

```

Comment:
```suggestion
    name: "ComfyUI-Image-Saver",
    settings: [
        {
            id: "ImageSaver.HandleImageWorkflowDrop",
            name: "Load workflows from dropped JPEG and WEBP files",
            type: "boolean",
            defaultValue: true
        },
    ],
```
Let's make it configurable, to avoid double loading in case user has another extension doing this already.